src/mds/mdstypes.cc

```diff
@@ -444,25 +448,45 @@ feature_bitset_t::feature_bitset_t(unsigned long value)
   }
 }
 
-feature_bitset_t::feature_bitset_t(const vector<size_t>& array)
+void feature_bitset_t::init_array(const vector<size_t>& array)
 {
   if (!array.empty()) {
     size_t n = array.back();
-    n += bits_per_block;
-    n /= bits_per_block;
-    _vec.resize(n, 0);
+     n += bits_per_block;
```

Comment:
nit: something's wrong with the indentation in this function

Reply:
Yes, I'll fix it.

Reply:
Done.